lib/plugin/allure.js

```diff
@@ -215,7 +215,13 @@ module.exports = (config) => {
     }
 
     err.message = err.message.replace(ansiRegExp(), '');
-    reporter.endCase('failed', err);
+    if (reporter.getCurrentTest()) {
+      reporter.endCase('failed', err);
+    } else if (err) {
+      // this means before suite failed, we should report this.
```

Comment:
This fixes a broken case t the moment where if a `BeforeSuite` fails there is nothing in the allure report indicate a problem. This is especially problematic when running parallel. 

I wasn't sure it it was possible to generate name here rather than hard coding a blank test case.
'Suite setup failed'.

The result here is that:
![image](https://user-images.githubusercontent.com/9056958/75786507-55243780-5d5d-11ea-8212-2ae75d86cc02.png)


How does this impact run-workers where `BeforeSuite` can be run multiple time? see below
The setup failure is correctly reported: 
![image](https://user-images.githubusercontent.com/9056958/75786403-24dc9900-5d5d-11ea-8365-fd5dd97ba2f6.png)
(WRONG NUMBER HERE error message comes from my test where i'm randomly generating number between 0 and 1 so i can fail the beforesuite in one or more threads)